autoload/neomake.vim

```diff
@@ -368,9 +371,9 @@ function! s:AddExprCallback(maker) abort
 
     if list_modified
         if file_mode
-            call setloclist(a:maker.winnr, list, 'r')
+            call setloclist(a:maker.winnr, list, 'r', name)
         else
-            call setqflist(list, 'r')
+            call setqflist(list, 'r', name)
```

Comment:
Please handle `E118` for backwards compatibility: https://github.com/neovim/neovim/pull/4337#issuecomment-188891314 (maybe wrap/refactor it in a function, because there are 2/4 places).